### PR TITLE
Fix: Changed start button to resume during gameplay

### PR DESCRIPTION
**Pull request by top-gun007**
PR for #279 
image

For the above change I made the following changes:
- added resume text in strings.xml.

- set`android:textStyle="bold"`to bold in activity_main.xml

- used `boolean hasPreviouslyStarted` to determine weather gameplay has started or not.

### CODE REVIEW

**Comment by anubhakushwaha** [comment]
+1

**Comment by top-gun007** [comment]
@anubhakushwaha thank you.
Can you merge it please since I can only work on my second issue once this is merged(changes to be done in the same file).

**Comment by anubhakushwaha** [comment]
@chhavip I think it can be merged.

File: PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java [hunk 1]
```diff
@@ -18,22 +19,23 @@
 
     private SharedPreferences preferences;
     private boolean hasPreviouslyStarted;
+    Button startButton, newUserButton, aboutButton;
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java]: Declare them separately and as private

Comment (top-gun007) [inline PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java]: on it!

**Comment by top-gun007** [comment]
@chhavip requested changes done.

**Comment by top-gun007** [comment]
Please merge @chhavip

**Comment by chhavip** [comment]
Merging. Looks good.

**Pull request merged**